config/defaults.ini

```diff
@@ -502,7 +503,7 @@ apps_add =
   org.gimp.GIMP
   org.gnome.Cheese
   org.gnome.Gnote
-  org.gnome.Rhythmbox3
+  org.gnome.Music
```

Comment:
Very convenient that they fall in the same place in the sort order!

Reply:
Indeed!